astrosat_users/forms/forms_auth.py

```diff
@@ -64,10 +64,10 @@ def save(self, request, **kwargs):
 
 class RegistrationForm(AllAuthRegistrationForm):
 
-    field_order = ["email", "password1", "password2", "requires_customer_registration_completion", "accepted_terms"]
+    field_order = ["email", "password1", "password2", "registration_stage", "accepted_terms"]
 
     accepted_terms = forms.BooleanField(label="Accept Terms & Conditions")
-    requires_customer_registration_completion = forms.BooleanField(label="User is Registering as part of a Team", required=False)
+    registration_stage = forms.CharField(label="What stage of the registration process is the user at", required=False)
```

Comment:
Would this not be better as an `enum`? I assume there will only be a few possible stages.

Reply:
The field as defined in the model already uses an enum.  Therefore, validation on this form should fail if anything other than one of the predefined choices is provided.